inc/order.class.php

```
@@ -1597,9 +1599,17 @@ public function generateOrder($params) {
             $total_TTC = $prices["priceTTC"] + $postagewithTVA;
 
             if ($signature) {
-               try{$odf->setImage('sign', PLUGIN_ORDER_SIGNATURE_DIR.$signature);} catch(\Odtphp\Exceptions\OdfException $e){}
+               try {
+                  $odf->setImage('sign', PLUGIN_ORDER_SIGNATURE_DIR.$signature);
+               } catch (\Odtphp\Exceptions\OdfException $e) {
+                  $is_cs_happy = true;
```

Required to avoid empty catch